tests/unit/testServiceConfiguration.js

```diff
@@ -24,546 +24,546 @@ var logger = require('../../src/logger.js')({
-        }
-      }, logger);
-      t.deepEqual(c.getServiceContext().service, 'customService');
-      // The user specified version should be used
-      t.deepEqual(c.getServiceContext().version, '2.0');
-
-      end();
-  })
-);
-
-test(
-  'With GAE_SERVICE and GAE_VERSION not set: ' +
-  'A Configuration uses the service name "node" and no version if ' +
-  'GAE_MODULE_NAME is not set, FUNCTION_NAME is not set, and the user has '+
-  'not specified a service name or version',
-  makeTest(null, null, null, null, null,
-    function(t, end) {
-      var c = new Configuration({}, logger);
-      t.deepEqual(c.getServiceContext().service, 'node');
-      t.deepEqual(c.getServiceContext().version, undefined);
-
-      end();
-  })
-);
-
-test(
-  'With GAE_SERVICE and GAE_VERSION not set: ' +
-  'A Configuration uses the service name "node" and no version if ' +
-  'GAE_MODULE_NAME is not set, FUNCTION_NAME is not set, and the user has '+
-  'not specified a service name or version even if GAE_MODULE_VERSION has ' +
-  'been set',
-  makeTest(null, null, null, '1.0', null,
-    function(t, end) {
-      var c = new Configuration({}, logger);
-      t.deepEqual(c.getServiceContext().service, 'node');
-      t.deepEqual(c.getServiceContext().version, undefined);
-
-      end();
-  })
-);
-
-test(
-  'With GAE_SERVICE and GAE_VERSION not set: ' +
-  'A Configuration uses the service name "node" and the user specified ' +
-  'version if GAE_MODULE_NAME is not set, FUNCTION_NAME is not set, and the ' +
-  'user has not specified a service name but has specified a version',
-  makeTest(null, null, null, null, null,
-    function(t, end) {
-      var c = new Configuration({
-        serviceContext: {
-          version: '2.0'
-        }
-      }, logger);
-      t.deepEqual(c.getServiceContext().service, 'node');
-      t.deepEqual(c.getServiceContext().version, '2.0');
-
-      end();
-  })
-);
-
-test(
-  'With GAE_SERVICE and GAE_VERSION not set: ' +
-  'A Configuration uses the service name "node" and the user specified ' +
-  'version if GAE_MODULE_NAME is not set, FUNCTION_NAME is not set, and the ' +
-  'user has not specified a service name but has specified a version even ' +
-  'if GAE_MODULE_VERSION has been set',
-  makeTest(null, null, null, '1.0', null,
-    function(t, end) {
-      var c = new Configuration({
-        serviceContext: {
-          version: '2.0'
-        }
-      }, logger);
-      t.deepEqual(c.getServiceContext().service, 'node');
-      t.deepEqual(c.getServiceContext().version, '2.0');
+      // setup the new environment
+      updateEnv('GAE_SERVICE', serviceName);
+      updateEnv('GAE_VERSION', serviceVersion);
 
-      end();
-  })
-);
+      updateEnv('GAE_MODULE_NAME', moduleName);
+      updateEnv('GAE_MODULE_VERSION', moduleVersion);
+
+      updateEnv('FUNCTION_NAME', functionName);
+
+      // run the function
+      return testFunction(done ,function() {
+        process.env.GAE_SERVICE = oldServiceName;
```

Comment:
From how it looks, it seems like the call to this function was lost in translation, since functions passed as the `testFunction` argument of `makeTest` no longer make a call to the function provided as their second argument. What's in this function should be moved to an `afterEach` clause.